### PR TITLE
add test build step for windows

### PR DESCRIPTION
**Pull request by linhkikuchi**
Closes https://github.com/brave/devops/issues/1316

## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/devops/issues/1316) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [x] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [x] Windows
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by linhkikuchi** [comment]
reformat the try/catch block script to shorten the Jenkinsfile, jenkins has issue with large Jenkinsfile, it will hit `Method code too large issue` if we do not reformat it

File: Jenkinsfile [hunk 1]
```diff
@@ -890,17 +810,24 @@ pipeline {
                                 """
                             }
                         }
+                        stage("test-install-x64") {
+                            steps {
+                                timeout(time: 5, unit: "MINUTES") {
+                                    catchError(buildResult: 'UNSTABLE', stageResult: 'FAILURE') {
+                                        script {
+                                            testInstallWins()
```
Comment (mihaiplesa) [inline Jenkinsfile]: `testInstallWindows` should do

File: Jenkinsfile [hunk 2]
```diff
@@ -890,17 +810,24 @@ pipeline {
                                 """
                             }
                         }
+                        stage("test-install-x64") {
```
Comment (mihaiplesa) [inline Jenkinsfile]: as above, no need for `x64`

File: Jenkinsfile [hunk 3]
```diff
@@ -1103,3 +1030,25 @@ def printException(ex) {
...
+        # make sure there are brave process
+        \$Service = Get-Process | Where {\$_.ProcessName -eq "brave"}
+        If (\$Service) { "Brave Browser was installed and running"  }
+        Else { 
+            "Brave Browser was not running" 
+            exit 1
+        }
+        # Stop brave
+        Stop-Process -Name "Brave*" -Force
+        rm -r "C:\\Users\\Administrator\\Desktop\\Brave*"
```
Comment (mihaiplesa) [inline Jenkinsfile]: would suggest using `Remove-Item -Recurse -Force` as at https://github.com/brave/brave-browser/blob/master/Jenkinsfile#L753

**Pull request merged**